appinventor/components/src/com/google/appinventor/components/runtime/LineChartModel.java

```diff
@@ -22,6 +25,11 @@ public LineChartModel(LineData data) {
         this.data.addDataSet(dataset); // Safe add
     }
 
+    @Override
+    protected int getTupleSize() {
+        return 2;
```

Comment:
Add TBD issue to add abstractions for this (as this applies for quite a lot of Chart types)